summerofcode/2023.md

```diff
@@ -129,6 +129,31 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
 - Difficulty: Hard
 - Upstream Issues (URL): https://github.com/kubevela/kubevela/issues/5366
 
+#### Auto-generate the TypeScript and Java languages API SDK
+
+- Description: Some developers start to integrate KubeVela into the enterprise system. it is increasingly important to provide API SDK to help developers start faster. Java and TypeScript are currently the two most in-demand languages. KubeVela API server follows the Open API schema, It means the Open API generator tools could provide some help.
```

Comment:
```suggestion
- Description: Some developers have started to integrate KubeVela into the enterprise system. It is getting increasingly important to provide an API SDK to help developers start faster. Java and TypeScript are currently the two most in-demand languages. KubeVela API server follows the Open API schema. It means the Open API generator tools could provide some help.
```

Optional grammar fixes, sorry for nit picking 🙏

Reply:
Thanks!